packages/packages/store/src/__tests__/store.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
```

Comment:
Maybe add some tests for the single instance:
when trying to dispatch without an instance of a store
when trying to recreate a new instance of a store

Reply:
This test: "when trying to dispatch without an instance of a store" - already exist.